prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,10 @@
+const array_name=["html","css","javascript","react","node","postgres"]
+console.log(array_name)
+var firstElement= "html"
+console.log(firstElement)
+var thirdelement= "javascript"
+console.log(thirdelement)
+var length = 6
+console.log(length)
+var lastelement= "postgres"
+console.log(lastelement)
```

Comment:
In a lot of situations, we don't know how long an array is (or will be), so we can grab the last element of the array by subtracting `1` from its `length`

```js
var lastElement = array_name[array_name.length - 1];
```